packages/clay-css/src/scss/components/_cards.scss

```diff
@@ -18,15 +18,13 @@
 	position: relative;
 	word-wrap: break-word;
 
-	@if ($card-border-radius > 0) {
-		.autofit-col {
-			&:first-child {
-				@include border-left-radius($card-inner-border-radius);
-			}
+	.autofit-col {
+		&:first-child {
+			@include border-left-radius($card-inner-border-radius);
+		}
 
-			&:last-child {
-				@include border-right-radius($card-inner-border-radius);
-			}
+		&:last-child {
+			@include border-right-radius($card-inner-border-radius);
```

Comment:
@pat270 the only side effect this adds is that it will create the classes even when the `border-radius` is zero right?

Reply:
@matuzalemsteles yes, I tried to save some CSS a long time ago. It doesn't really work well.